### ISSUE TITLE
var application = require("application"); crashes on ios

### ISSUE THREAD

**Issue by fumpierrez** [nativescript-vue/nativescript-vue#236]
### Version
1.3.1

### Reproduction link
[https://play.nativescript.org/?template=play-vue&id=NYSiDG](https://play.nativescript.org/?template=play-vue&id=NYSiDG)
### Platform and OS info
IOS 6/ NativeScript 4.0.2/ vue-cli 2.9.3

### Steps to reproduce
Create new project: vue init nativescript-vue/vue-cli-template nstest

Edit src/main.js  and simply add:

var application = require("application");

npm run watch:ios will error with:

file:///app/app.js:1:136738: JS ERROR Error: A Frame must be used to navigate to a Page.

However, android works well

### What is expected?
No error

### What is actually happening?
Not sure, I think there might be a bug in the application class when running on ios

---
Need to load application so I can set a delegate for ios

**Comment by rigor789** [comment]
I ran the playground but it seems to be working on iOS. Do you have a github repo that I can run locally that demonstates the issue? 

You may need to try the alpha release see https://github.com/nativescript-vue/nativescript-vue/issues/239#issuecomment-394780299 for details on how to do it.

**Comment by fumpierrez** [comment]
@rigor789 https://github.com/fumpierrez/nativescript-vuejs-bug-example

That's a repo with code that fails on my machine (tried 2 different computers)

I'll try bumping to the alpha version as per your link

**Comment by slimzc** [comment]
Hi, I had the same problem, but when I add a alias in my webpack.config.js the problem was solved.

just like you see in this demo (Grocery Demo using .vue template): https://github.com/tralves/groceries-ns-vue/blob/master/webpack.config.js#L97

**Comment by uniquejava** [comment]
I have exactly the same issue, I described my steps here https://github.com/nativescript-vue/nativescript-vue/issues/255
@slimzc  Your method does not work for me.

**Comment by fumpierrez** [comment]
neither did for me

**Comment by msaelices** [comment]
@fumpierrez I tested your repo with the last NS-vue `2.0.0-alpha.1` version and it worked well on both platforms.

See https://github.com/fumpierrez/nativescript-vuejs-bug-example/pull/1

I think one of the possible issues is to have `vue-router` included in the app, as this integration is experimental until 2.0.0 was finally released.

BTW, I recommend you change the way of importing the application, like that:

```
import * as app from 'tns-core-modules/application'
```
or
```
application = require('tns-core-modules/application')
```

The important thing here is to have the `tns-core-modules/` prefix in your import sentence.
